demos/smart_classroom_demo/cpp_gapi/include/initialize.hpp

```diff
@@ -327,6 +327,7 @@ std::shared_ptr<FaceRecognizer> processingFaceGallery(const cv::gapi::GNetPackag
         int id = 0;
         for (const auto& item : fn) {
             std::string label = item.name();
+            std::cout << label << std::endl;
```

Comment:
is it required? Seems like an unstructured random output

Reply:
No. Removed.